src/main.ts

```diff
@@ -247,7 +247,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
     vscode.commands.registerCommand('latex-workshop.showCompilationPanel', () => extension.buildInfo.showPanel())
 
-    context.subscriptions.push(vscode.commands.registerCommand('latex-workshop.formattedPaste', () => extension.paster.paste()))
+    context.subscriptions.push(vscode.commands.registerCommand('latex-workshop.paste', () => extension.paster.paste()))
```

Comment:
merge

Reply:
Did in 4af193574c0483b73c57ede6387cd92e092bbecf